Solutions/FalconFriday/Analytic Rules/Component Object Model Hijacking.yaml

```diff
@@ -0,0 +1,46 @@
+id: 1aaff41f-4e18-45b1-bb34-de6eb4943cf2
+name: Component Object Model Hijacking - Vault7 trick
+description: |
+  This detection looks for the very specific value of "Attribute" in the "ShellFolder" CLSID of a COM object. This value (0xf090013d) seems to only link back to this specific persistence method. 
+  The blog post linked here (https://www.ired.team/offensive-security/code-execution/forcing-iexplore.exe-to-load-a-malicious-dll-via-com-abuse) provides more background on the meaning of this value. 
+severity: Medium
+requiredDataConnectors:
+  - connectorId: MicrosoftThreatProtection
+    dataTypes:
+      - DeviceRegistryEvents
+queryFrequency: 1h
+queryPeriod: 1h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+  - PrivilegeEscalation
+relevantTechniques:
+  - T1546.015
+query: |
+  DeviceRegistryEvents
+  | where RegistryKey contains "ShellFolder"  and ActionType == "RegistryValueSet" and RegistryValueName =~ "Attributes" 
```

Comment:
- replace `contains` with `has` also for ActionTypes replace `==` with `=~`
- consider projecting selected columns to the output rather than displaying all fields

Reply:
Replaced `contains` with `has`
Replaced `==` with `=~`.
I'd like this rule to give the user full background, without projecting anything. Hope that's oke.